src/sbp.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '@sbp/sbp' {
```

Comment:
A note about this file: this is no longer needed once `sbp` is a TypeScript module. I'll double check that `sbp` is exporting types consistent with these definitions.